Hono RPC client response missing body

### What version of Hono are you using?

3.11.10

### What runtime/platform is your app running on?

bun

### What steps can reproduce the bug?

https://github.com/lawrencecchen/hono-hc-no-body

TypeScript error in https://github.com/lawrencecchen/hono-hc-no-body/blob/main/client.ts

```bash
client.ts:8:25 - error TS2339: Property 'body' does not exist on type 'ClientResponse<{}>'.

8 const reader = response.body.getReader();
                          ~~~~


Found 1 error in client.ts:8
```

Code:

```ts
// client.ts
import { hc } from "hono/client";
import { app } from ".";

const client = hc<typeof app>("http://localhost:3000");
console.log("fetching...");
const response = await client.api.awesome.$get();
// no body sad :(
const reader = response.body.getReader();
const decoder = new TextDecoder();
while (true) {
	const { done, value } = await reader.read();
	if (done) break;
	const decoded = decoder.decode(value);
	console.log(decoded);
}
```

```ts
// index.ts
import { Hono } from "hono";

export const app = new Hono()
	.get("/", (c) => c.text("root"))
	.get("/api/awesome", async (c) => {
		return c.streamText(async (stream) => {
			stream.writeln("Hello World!");
			await new Promise((r) => setTimeout(r, 1000));
			stream.writeln("Very cool!");
			await stream.close();
		});
	});

export default app;
```

### What is the expected behavior?

Response should have body

### What do you see instead?

No body

### Additional information

My use case is streaming

--- a/deno_dist/client/types.ts
+++ b/deno_dist/client/types.ts
@@ -30,6 +30,8 @@ export type ClientRequest<S extends Schema> = {
 type BlankRecordToNever<T> = T extends any ? (keyof T extends never ? never : T) : never
 
 export interface ClientResponse<T> {
+  readonly body: ReadableStream | null
+  readonly bodyUsed: boolean
   ok: boolean
   status: number
   statusText: string

--- a/src/client/types.ts
+++ b/src/client/types.ts
@@ -30,6 +30,8 @@ export type ClientRequest<S extends Schema> = {
 type BlankRecordToNever<T> = T extends any ? (keyof T extends never ? never : T) : never
 
 export interface ClientResponse<T> {
+  readonly body: ReadableStream | null
+  readonly bodyUsed: boolean
   ok: boolean
   status: number
   statusText: string

